### ISSUE TITLE
tinygo error

### ISSUE THREAD

**Issue by davmaz** [johanbrandhorst/wasm-experiments#12]
Thanks for the great overview. However, when I try make tinygo, I get this error:
...
Digest: sha256:a07366ee23684c07276f3a2d73a70f4bd00bfb0afe6ee229d0f35cf76ddc386d
Status: Downloaded newer image for tinygo/tinygo:0.8.0
error: cannot find package "github.com/NYTimes/gziphandler" in any of:
	/usr/local/go/src/github.com/NYTimes/gziphandler (from $GOROOT)
	/root/go/src/github.com/NYTimes/gziphandler (from $GOPATH)
Makefile:9: recipe for target 'tinygo' failed

I've tried removing the :0.8.0 and it still fails.

**Comment by johanbrandhorst** [comment]
Thanks for the issue, looks like that repo has moved to `github.com/nytimes/gziphandler`. Would you be interesting in contributing a fix for this? It should be a simple search replace.

**Comment by davmaz** [comment]
Where do I find the Dockerfile used to build tinygo/tinygo-dev ? That is what I need to update, right? I don't have much experience with docker (it's growing slowly) so please guide me in the right direction.

**Comment by johanbrandhorst** [comment]
The error is here: https://github.com/johanbrandhorst/wasm-experiments/blob/master/main.go#L9

**Comment by davmaz** [comment]
You are correct as github.com/NYTimes/gziphandler is the right package. However, I don't know how to 'fix' the docker image.

**Comment by johanbrandhorst** [comment]
I don't think it's an issue with the docker image, just the import. If you replace that line that I linked with the new import part it might work. Could you test it?

**Comment by taigrr** [comment]
@johanbrandhorst I tried to take a crack at solving this but I ran into a few errors that eventually pointed at a 'bug' (missing feature) in tinygo.

Go can resolve dependencies regardless of case, but tinygo seems to require the VCS url and package name use the same casing.

This issue is discussed here: https://github.com/nytimes/gziphandler/issues/78

Also here: https://github.com/nytimes/gziphandler/pull/82

I'm not yet familiar enough with the tinygo project to open a PR or issue there, but I just wanted to make it known the change can't be made in this repo without vendoring until either the nytimes repo bumps to the next major version or tinygo's package management reaches parity with that of go's.
